src/main/java/Duke.java

```diff
@@ -6,5 +12,49 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n");
+        String line;
+        List<Task> tasks = new ArrayList<>();
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        while(!line.equals("bye")) {
+            System.out.println("____________________________________________________________\n");
+            if (line.equals("list")) {
+                for (int i = 0; i < tasks.size(); i++) {
+                    Task todoList = tasks.get(i);
+                    System.out.println("\t" + (i+1) + todoList.getStatusIcon() + todoList.description);
+                }
+                System.out.println("____________________________________________________________\n");
+            }
+            else if (line.contains("unmark")) {
+                int index = line.indexOf("") + 1;
+                Task taskUnmarked = tasks.get(index - 1);
+                taskUnmarked.markUndone();
```

Comment:
good job on making the code looks tidy, proper indentation and spacing, the naming of the variable is also proper, and the code is very intuitive and easy to read